src/pages/Settings.js

```diff
@@ -52,14 +52,22 @@ export default function SettingsScreen({ navigation }) {
 
           let ec1n = emergencyContact1Name
           let ec1p = emergencyContact1Phone
-          let ec2n = emergencyContact1Name
-          let ec2p = emergencyContact1Phone
-          let ec3n = emergencyContact1Name
-          let ec3p = emergencyContact1Phone
-          let ec4n = emergencyContact1Name
-          let ec4p = emergencyContact1Phone
-          let ec5n = emergencyContact1Name
-          let ec5p = emergencyContact1Phone
+          let ec2n = emergencyContact2Name
+          let ec2p = emergencyContact2Phone
+          let ec3n = emergencyContact3Name
+          let ec3p = emergencyContact3Phone
+          let ec4n = emergencyContact4Name
+          let ec4p = emergencyContact4Phone
+          let ec5n = emergencyContact5Name
+          let ec5p = emergencyContact5Phone
```

Comment:
So I know absolutely nothing about js but why this and not just put the variables straight into the arrays?

Reply:
I was having some issues with that and this was just the dumb solution

Reply:
Ah nice, as long as it works it doesn't have to look pretty